src/main/java/com/palantir/stash/stashbot/managers/PluginUserManager.java

```diff
@@ -56,7 +56,7 @@ public PluginUserManager(UserAdminService uas, PermissionAdminService pas, UserS
     }
 
     public void createStashbotUser(JenkinsServerConfiguration jsc) {
-        StashUser user = us.getUserByName(jsc.getStashUsername());
+        ApplicationUser user = us.getUserByName(jsc.getStashUsername());
```

Comment:
Yeah, maybe I'll fix that in a future change.  Good point, would be less confusing.